portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ * Adds a random activity car to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Gone Now by Bleachers', 'Malibu Nights by LANY', 'Melodrama by Lorde','After Hours by The Weeknd', 'American Teen by Khalid'];
+function addRandomActivity() {
+  const activityTitles = [
+    "Bake Banana Bread!",
+    "Listen to an Album!",
+    "Watch a RomCom!",
+    "Take a Nap!",
+    "Read a Book!"
+  ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const activityDescriptions = [
+    "You've seen it all over instagram, now is the time to learn to bake banana bread! I recommend Bon Appetit's recipe(linked below), which I have been guilty of making multiple times in a week(enjoy the picture of my latest loaf below!).",
+    "Listen to an entire album. Yes, I mean start to finish! When will you have the chance to after this? Some recommendations include 'Fine Line' by Harry Styles, 'After Hours' by The Weeknd, and 'Melodrama' by Lorde.",
+    "A personal favorite choice, now is the time to start a group call with friends and watch (or rewatch!) a romcom. Some recommendations include 'The Half of It,' 'Someone Great,' 'Clueless,' and '10 Things I Hate About You.'",
+    "zzz",
+    "Yes, a real book, doesn't matter if it's paperback, hardcover, or an ebook. When was the last time you read one? I'm personally taking the time to reread the Harry Potter series.",
+  ];
 
+  const activityPhotos = [
```

Comment:
I would call them "Images" instead of photos. Here and everywhere below.

Reply:
Also, how about "activityImageFilenames" since these are actually the names of the files and not the images themselves.